### ISSUE TITLE
Allow writing `boolean` in schema for `valueType` 

### ISSUE THREAD

**Issue by CJ42** [ERC725Alliance/erc725.js#268]
### I'm submitting a...

- [ ] bug report
- [x] feature request
- [ ] question about the decisions made in the repository
- [ ] question about how to use this project

In LSP2, the `valueType` for boolean values is `bool` because the type for true/false values in Solidity is declared as `bool`.

However, most user of the _erc725.js_ library are Javascript developers. In Javascript the type is `boolean`.

It is also more common in plain english to use the terminology "boolean" (We say in plain english "a boolean value" ✅, not "a bool value" ❌).

Allow in erc725.js to use the word "`boolean`" in `valueType`. This should fallback by default to encoding/decoding as `bool`.

**Comment by Hugoo** [comment]
@CJ42 I think this was closed in PR: https://github.com/ERC725Alliance/erc725.js/pull/266

?

Can we close?

**Comment by CJ42** [comment]
@Hugoo yes this was implemented ✅ 
Can be closed.
